src/stack-hci/setup.py

```diff
@@ -12,7 +12,7 @@
 # HISTORY.rst entry.
 VERSION = '0.1.4'
 try:
-    from .manual.version import VERSION
+    from azext_stack_hci.manual.version import VERSION
```

Comment:
Is this change generated by codegen automatically?

Reply:
yes, it is.